File: moviejava/app/src/main/java/com/example/moviejava/model/JsonObject.java
package com.example.moviejava.model;


import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class JsonObject {
    @SerializedName("page")
    public int page;
    @SerializedName("results")
    public ArrayList<Results> results;
    @SerializedName("total_pages")
    public int totalPages;
    @SerializedName("total_results")
    public int totalResults;

}

File: movie/app/src/main/java/com/example/movie/model/JsonObject.kt
package com.example.movie.model


import com.google.gson.annotations.SerializedName

data class JsonObject(
    @SerializedName("page") var page: Int? = null,
    @SerializedName("results") var results: ArrayList<Results> = arrayListOf(),
    @SerializedName("total_pages") var totalPages: Int? = null,
    @SerializedName("total_results") var totalResults: Int? = null

)